.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "RobustToolbox"]
 	path = RobustToolbox
-	url = https://github.com/space-wizards/RobustToolbox.git
+	url = https://github.com/PrPleGoo/RobustToolbox.git
```

Comment:
Just so you know, you don't need to modify this file to check out your fork. Once the submodule is initialized you can run the normal `git remote` commands in it and it won't affect the master repo.